development/tools/jobs/kyma/docsbuilder_test.go

```diff
@@ -17,7 +17,7 @@ func TestDocsBuilderJobPresubmit(t *testing.T) {
 	assert.Len(t, jobConfig.Presubmits, 1)
 	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
 	assert.True(t, ex)
-	assert.Len(t, kymaPresubmits, 1)
+	assert.Len(t, kymaPresubmits, 2)
```

Comment:
you should update RELEASE_VERSION, otherwise you are going to release version 0.6.2

Reply:
done